databases/week2/Part4.sql

```diff
@@ -0,0 +1,31 @@
+CREATE database series_hyf;
+
+use series_hyf;
+
+CREATE TABLE actor(
+`id` int unsigned auto_increment NOT NULL PRIMARY KEY,
+`name` varchar(255) NOT NULL
+);
+
+CREATE TABLE category(
+`id` int unsigned auto_increment NOT NULL PRIMARY KEY,
+`name` varchar(255) NOT NULL
+);
+
+CREATE TABLE series(
+`id` int unsigned auto_increment NOT NULL PRIMARY KEY,
+`title` varchar(255) NOT NULL,
+`release_date` datetime NOT NULL,
+`end_date` datetime NOT NULL,
```

Comment:
I would probably just use `date` for these two

Reply:
Thank you for all your feedback. I will make the changes.